src/components/NameListContext.js

```diff
@@ -52,6 +62,44 @@ class NameListProvider extends React.Component {
     callNames();
   };
 
+  // Calls the Popularity Chart Data
+  callChart = (selectedName, selectedSex) => {
+    const chartUrl = `https://baby-namer-api.herokuapp.com/names?name=${selectedName}&sex=${selectedSex}`;
+    const constructChart = async () => {
```

Comment:
It would be good to have a single try/catch for the fetch operation, and a second one if needed for chart construction. Is chart construction something we have to worry about failing?